src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Controller.cs

```diff
@@ -142,7 +124,7 @@ private void Retrigger()
                 return;
             }
 
-            sessionOpt.ComputeModel(GetProviders(), new SignatureHelpTriggerInfo(SignatureHelpTriggerReason.RetriggerCommand));
```

Comment:
what's the difference between 'retrigger' and 'update'?  'retrigger' sounds like it would trigger sig help if it wasn't already up, 'update' sounds like it would just update sig help if it was already up.  Is that not the case?  Can you explain what the terminology means to help keep things straight?

Thanks!